collectors/python.d.plugin/exim/README.md

```diff
@@ -3,6 +3,25 @@
 Simple module executing `exim -bpc` to grab exim queue.
 This command can take a lot of time to finish its execution thus it is not recommended to run it every second.
 
+## Requirements
+
+The module uses the `exim` binary, which can only be executed as root by default. We need to allow other users to `exim` binary. We solve that adding `queue_list_requires_admin` statement in exim configuration and set to `false`, because is `true` by default. On many Linux distributions, the default location of `exim` configuration is in `/etc/exim.conf`.
```

Comment:
"because **it** is `true` by default."

Reply:
@shortpatti this is changed and commited. Thank you.